crates/bevy_ui/src/node_bundles.rs

```diff
@@ -23,6 +23,10 @@ pub struct NodeBundle {
     pub style: Style,
     /// The background color, which serves as a "fill" for this node
     pub background_color: BackgroundColor,
+    /// How to draw the node's border
+    pub border_style: BorderStyle,
```

Comment:
Don't think this needs to be blocking, but would note that HTML supports different colors and styles of border for each side (e.g. you can have `10px solid red` for the left border and `20px dashed black` for the top border). This complicates drawing though as you then need a triangular chamfer at the point where the borders meet.

Reply:
Yes can't really draw anything except rectangles at the moment with the current way the UI renderer works, otherwise I'd have at least added different coloured edges.

Reply:
Just finishing this thing up, and realised multi-coloured border support isn't a problem. 
The UI's shader can draw trapezoids. We would just need an enum like in #8310 so that ExtractedUiNode could support the alternative geometry. It's not something I want to tackle in this PR but it's not difficult to implement if we don't have a shader-based solution ready in time for 0.11